store/storetest/channel_store.go

```diff
@@ -655,7 +655,9 @@ func testChannelStoreGetByName(t *testing.T, ss store.Store) {
 	result = <-ss.Channel().GetByName(o1.TeamId, "", false)
 	require.NotNil(t, result.Err, "Missing id should have failed")
 
-	store.Must(ss.Channel().Delete(channelID, model.GetMillis()))
+	err := ss.Channel().Delete(channelID, model.GetMillis())
+	require.Nil(t, err, "Channel must have been deleted")
```

Comment:
nit: s/must/should  (and for the 'must's below) -- just to make it more understandable when reading the report

Reply:
changed the same.. 👍